### DISCUSSION TITLE
Capstone 23/05 4week

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/23_1_Capstone#53] [General]
적으세요

**Comment by fkdl0048** [comment]
상점 NPC, UI 제작하여 연결하기  
기본적인 맵 아이콘, 콜라이더 작업
퀼리티 작업,,

**Comment by aquamagic9** [comment]
집꾸미기 내집 배경과 연결
chatGPT 퀘스트 구현하기
NPC 말풍선 구현

**Comment by LJI028** [comment]
개인 콘텐츠 서버 구현
애니메이션 등 세부 사항 수정

**Comment by bbabbungtting** [comment]
맵 세부 사항 구현

**Comment by ThugMS** [comment]
MainMenu에서 InGame으로 전환되면서 서버 연결
OX퀴즈와 집 방문 최적화
